thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/translator/DetectionConfigTranslator.java

```diff
@@ -189,6 +190,7 @@ private DetectionConfigDTO generateDetectionConfig(Map<String, Object> yamlConfi
     config.setCron(cron);
     config.setActive(MapUtils.getBooleanValue(yamlConfigMap, PROP_ACTIVE, true));
     config.setYaml(yamlConfig);
+    config.setBackfillStart(parseTimeStampLong(yamlConfigMap.get(PROP_BACKFILL_START)));
```

Comment:
I think we already have a utility for this. Can you try MapUtils or ConfigUtils?

Reply:
I found `org.apache.pinot.thirdeye.detection.ConfigUtils#getLongs(java.util.Collection<java.lang.Number>)` but that takes a list and has a slightly different behavior which would require me to have a separate method anyway.

Are you aware of any other utility? Please do share if you find something.